website/docs/r/is_subnet.html.markdown

```diff
@@ -39,6 +39,31 @@ resource "ibm_is_subnet" "testacc_subnet" {
 }
 ```
 
+## Example usage with address prefix
+```terraform
+resource "ibm_is_vpc" "testacc_vpc2" {
+  name = "test"
```

Comment:
if we are creating address prefix then should we set address_prefix_management = "auto"